src/views/liste-aides.vue

```diff
@@ -131,10 +148,11 @@ async function computeDataSelected() {
     selectedCommune.value = res[0]
   } else {
     selectedCommune.value = null
+    searchTerms.value = zipCode.value.toLowerCase()
   }
 }
 watch(zipCode, (newZipCode: string) => {
-  if ([0, 5].includes(newZipCode.length)) {
+  if ([0, 3].includes(newZipCode.length)) {
```

Comment:
Ce n'est pas vraiment ce qui est attendu dans une recherche par mot clef, ici la recherche sera déclenché si la barre contient soit 0 soit 3 caractères. J'aurais tendance à proposer ceci :

```suggestion
  if (newZipCode.length > 0) {
```

Reply:
> Ce n'est pas vraiment ce qui est attendu dans une recherche par mots-clés, ici la recherche sera déclenchée si l'input contient soit 0, soit 3 caractères. J'aurais tendance à proposer ceci :

Bien vu, j'ai améliorer ta recherche en mode > 2, c'est une recommendation du RGESN (normalement > 3) mais bon pour FSL c'est trois caractères.

Reply:
Ok nickel la reco !